substrate/frame/tx-pause/src/mock.rs

```diff
@@ -97,18 +100,28 @@ impl pallet_proxy::Config for Test {
 	type RuntimeCall = RuntimeCall;
 	type Currency = Balances;
 	type ProxyType = ProxyType;
-	type ProxyDepositBase = ConstU64<1>;
-	type ProxyDepositFactor = ConstU64<1>;
 	type MaxProxies = ConstU32<4>;
 	type WeightInfo = ();
 	type CallHasher = BlakeTwo256;
 	type MaxPending = ConstU32<2>;
-	type AnnouncementDepositBase = ConstU64<1>;
-	type AnnouncementDepositFactor = ConstU64<1>;
+	type ProxyConsideration = HoldConsideration<
```

Comment:
A good way to prevent these changes is to introduce a `TestDefaultConfig` for `pallet_proxy` along the way :)

Reply:
Done